services/net/transcription/TranscriptionManager.cs

```diff
@@ -306,6 +307,22 @@ private async Task UpdateTranscriptionAsync(TranscriptRequestModel request, Cont
         }
     }
 
+    private static string GetFormattedTranscript(string transcript)
+    {
+        var result = transcript;
+        if (!string.IsNullOrWhiteSpace(result))
+        {
+            var pattern = @"\.[A-Z0-9]";
+            var matches = Regex.Matches(result, pattern);
+            foreach (Match match in matches)
+            {
+                result = result.Replace(match.Value, match.Value[0] + " " + match.Value[1]);
```

Comment:
Is there always a two position array result, or is this going to sometimes result in a runtime error?

Reply:
Yes there is always a two position array result in this case, and it seems to be working fine so far, thanks.